next/pages/api/articles-listing.ts

```diff
@@ -0,0 +1,42 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { DrupalNode } from "next-drupal";
+import { drupal } from "lib/drupal";
+
+import { validateAndCleanupArticleTeaser } from "@/lib/zod/article-teaser";
+
+import siteConfig from "@/site.config";
```

Comment:
This is a bit strange, but it seems to work. We need to get to the default locale, and it's configured in that file.

Reply:
Yeah, I think it works. We import it also in `language-links-context.tsx` and haven't seen any issues with it